packages/addons/service/tvheadend42/package.mk

```diff
@@ -2,16 +2,16 @@
 # Copyright (C) 2016-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="tvheadend42"
-PKG_VERSION="5613551"
-PKG_SHA256="eff2d5aa08d7c91595d2ec4d4dda2f1bedc07ee8672d1ffb5ed994e284714f23"
-PKG_VERSION_NUMBER="4.2.6-7"
-PKG_REV="115"
+PKG_VERSION="406ba887cfa7cee06c1e0f37cd45dbec3188abcd"
+PKG_SHA256="6670c4061f5869674beb525f157e6a66c05c2218bfff8114e15d66f94391554a"
+PKG_VERSION_NUMBER="4.2.6-42"
+PKG_REV="116"
 PKG_ARCH="any"
 PKG_LICENSE="GPL"
 PKG_SITE="http://www.tvheadend.org"
 PKG_URL="https://github.com/tvheadend/tvheadend/archive/$PKG_VERSION.tar.gz"
-PKG_SOURCE_DIR="tvheadend-${PKG_VERSION}*"
-PKG_DEPENDS_TARGET="toolchain avahi curl dvb-apps ffmpegx libdvbcsa libhdhomerun libiconv openssl pngquant:host Python2:host tvh-dtv-scan-tables"
+PKG_DEPENDS_TARGET="toolchain avahi comskip curl dvb-apps ffmpegx libdvbcsa libhdhomerun \
+                    libiconv openssl pngquant:host Python2:host tvh-dtv-scan-tables"
```

Comment:
Not sure continuing a variable assignment over multiple lines is a good idea, as this makes grepping for stuff that bit harder. A single line for these PKG_* variables is generally the "norm" and should be maintained IMHO.

Reply:
As discussed on Slack, leave it as it is and I'll merge this - we can decide what to about splitting in another PR (if necessary).